src/overlays/actors/ovl_En_Toryo/z_en_toryo.c

```diff
@@ -367,7 +367,7 @@ void EnToryo_Update(Actor* thisx, PlayState* play) {
         }
 
         rot = thisx->yawTowardsPlayer - thisx->shape.rot.y;
-        if ((rot < 14563.0f) && (rot > -14563.0f)) {
+        if ((rot < DEG_TO_BINANG2(80.0f)) && (rot > DEG_TO_BINANG2(-80.0f))) {
```

Comment:
how does this match in gc versions since the comparison is now s16 vs s32 compared to main's s16 vs f32 ?

Reply:
`rot` here is actually an f32, so it does a floating-point comparison in both cases, but with different float constants in N64 vs GC (14563.556f vs 14563.0f)